acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/SignerConfigurationBuilder.java

```diff
@@ -70,6 +71,11 @@ public SignerConfigurationBuilder withMetricsHostAllowList(final List<String> al
     return this;
   }
 
+  public SignerConfigurationBuilder withMetricsCategories(final List<String> metricsCategories) {
```

Comment:
is it worth making this an ellipsis - then do the conversion to a list in here (Saves a List.of everytime we use it)

Reply:
done